new_client/src/components/Header.js

```diff
@@ -1,21 +1,105 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
 
+import "./Header.css";
+export default function Header(props) {
+	const [isDesktop, setisDesktop] = useState(window.innerWidth > 1080);
+	const [navOpen, setnavOpen] = useState(false);
+	useEffect(() => {
+		function navToggle() {
+			setnavOpen((navPrevState) => {
+				return !navPrevState;
+			});
+		}
+		window.addEventListener("resize", () => {
+			setisDesktop(window.innerWidth > 1080);
+		});
+		document.getElementById("nav-toggler").addEventListener("click", navToggle);
```

Comment:
if we are adding an eventListener, we should ensure that we remove the eventListener when the component unmounts, in other words, return from useEffect `removeEventListener`.